SimCalorimetry/EcalEBTrigPrimAlgos/src/EcalEBPhase2Linearizer.cc

```diff
@@ -0,0 +1,166 @@
+#include <SimCalorimetry/EcalEBTrigPrimAlgos/interface/EcalEBPhase2Linearizer.h>
+
+//#include <CondFormats/EcalObjects/interface/EcalTPGCrystalStatus.h>
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+EcalEBPhase2Linearizer::EcalEBPhase2Linearizer(bool debug)
+    : debug_(debug), init_(false), peds_(nullptr), badXStatus_(nullptr) {}
+
+EcalEBPhase2Linearizer::~EcalEBPhase2Linearizer() {
+  if (init_) {
+    for (int i = 0; i < (int)vectorbadXStatus_.size(); i++) {
+      delete vectorbadXStatus_[i];
+    }
+  }
+}
+
+void EcalEBPhase2Linearizer::setParameters(EBDetId detId,
+                                           const EcalLiteDTUPedestalsMap *ecaltpPed,
+                                           const EcalEBPhase2TPGLinearizationConstMap *ecaltpLin,
+                                           const EcalTPGCrystalStatus *ecaltpBadX)
+
+{
+  EcalLiteDTUPedestalsMap::const_iterator itped = ecaltpPed->getMap().find(detId);
+  if (itped != ecaltpPed->end())
+    peds_ = &(*itped);
+  else
+    std::cout << " could not find EcalLiteDTUPedestal entry for " << detId << std::endl;
```

Comment:
This and the other `cout`'s in this class do not look as being protected by any `debug` flag

Reply:
Unlesse something is still missing ( not by choice) all couts in all these packages should be by now replaced by either LogError or Log Debug